distro/air-quality-distro.conf

```diff
@@ -0,0 +1,23 @@
+require conf/distro/poky.conf
+
+DISTRO = "air-quality-distro"
+
+DISTRO_NAME = "Distro for mini air quality station"
```

Comment:
Mini Air Quality Station OS?
It will be displayed when on login and in `/etc/issue` right?
Please consider

Reply:
Changed it now, thanks